wren-ai-service/Justfile

```
@@ -1,5 +1,37 @@
+up:
+	just prepare-wren-engine
+	docker compose -f ./tools/dev/docker-compose-dev.yaml --env-file ./tools/dev/.env.example up -d
```

Alternative, Just command also provides the dependencies for a command, refer to https://github.com/casey/just?tab=readme-ov-file#dependencies
```suggestion
up: prepare-wren-engine
	docker compose -f ./tools/dev/docker-compose-dev.yaml --env-file ./tools/dev/.env.example up -d
```